API/chat/tests.py

```diff
@@ -55,7 +62,8 @@ def post_and_get_response(self, text, timestamp, username, typing):
         """
         return self.client.post(
             reverse('chat:message', args=('channel', self.channel.name,)),
-            {'text': text, 'username': username, 'datetime_start': timestamp, 'typing': typing}
+            {'text': text, 'username': username, 'datetime_start': timestamp, 'typing': typing},
+            **{'HTTP_AUTHORIZATION': settings.PASS}
```

Comment:
Why do you unpack the extra arguments in this way? Just pass them as named arguments.